modules/mozillians/main.tf

```diff
@@ -1,7 +1,8 @@
 variable "environment" {}
 variable "vpc_id" {}
 variable "service_security_group_id" {}
-variable "elasticache_instance_size" {}
+variable "elasticache_redis_instance_size" {}
+variable "elasticache_memcached_instance_size" {}
```

Comment:
the module is namespaced so they'll always be referred to as `module.mozillians-staging.aws_elasticache_instance.foo` or something. I don't think it's worth adding redundancy.